x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.test.ts

```diff
@@ -99,6 +112,7 @@ const mockOptions = {
     ruleTypeId: '',
     ruleTypeName: '',
   },
+  logger,
```

Comment:
As far as I can see this executor still uses the logger injected via its `createMetricThresholdExecutor` function:

https://github.com/elastic/kibana/blob/6aae08871573202891c70824378ff9609dd94208/x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts#L73

Wouldn't it make sense to adapt that executor so it actually uses the new logger?